appveyor.yml

```diff
@@ -5,6 +5,10 @@ environment:
 
 install:
 
+cache:
+  - nsissetup.exe
+  - NsProcess.zip

```

Comment:
Another thing: I guess for this to work for nsissetup.exe and NsProcess.zip, you have to check if the files already exist in `after_test`, otherwise you'd just indiscriminately overwrite the cached versions again.